src/main/java/games/strategy/engine/message/RemoteMethodCall.java

```diff
@@ -87,51 +87,51 @@ public String getMethodName() {
   }
 
   private static Class<?>[] stringsToClasses(final String[] strings, final Object[] args) {
-    final Class<?>[] rVal = new Class<?>[strings.length];
+    final Class<?>[] classes = new Class<?>[strings.length];
     for (int i = 0; i < strings.length; i++) {
       try {
         // null if we skipped writing because the arg is the expected
         // class, this saves some space since generally the arg will
         // be of the correct type
         if (strings[i] == null) {
-          rVal[i] = args[i].getClass();
+          classes[i] = args[i].getClass();
         } else if (strings[i].equals("int")) {
-          rVal[i] = Integer.TYPE;
+          classes[i] = Integer.TYPE;
         } else if (strings[i].equals("short")) {
-          rVal[i] = Short.TYPE;
+          classes[i] = Short.TYPE;
         } else if (strings[i].equals("byte")) {
-          rVal[i] = Byte.TYPE;
+          classes[i] = Byte.TYPE;
         } else if (strings[i].equals("long")) {
-          rVal[i] = Long.TYPE;
+          classes[i] = Long.TYPE;
         } else if (strings[i].equals("float")) {
-          rVal[i] = Float.TYPE;
+          classes[i] = Float.TYPE;
         } else if (strings[i].equals("double")) {
-          rVal[i] = Double.TYPE;
+          classes[i] = Double.TYPE;
         } else if (strings[i].equals("boolean")) {
-          rVal[i] = Boolean.TYPE;
+          classes[i] = Boolean.TYPE;
         } else {
-          rVal[i] = Class.forName(strings[i]);
+          classes[i] = Class.forName(strings[i]);
```

Comment:
Although I think there must be a better way instead of just having tons of else if statements, a switch statement here would be perfect.